packages/example/lib/vision_detector_views/painters/barcode_detector_painter.dart

```diff
@@ -43,21 +43,21 @@ class BarcodeDetectorPainter extends CustomPainter {
       builder.pop();
 
       final left = translateX(
-        barcode.boundingBox.left,
+        barcode.boundingBox!.left.toDouble(),
```

Comment:
Avoid force unwrapping if you are uncertain or have not check if the property is not null

Reply:
ok